addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/BaseChannelHandler.java

```diff
@@ -104,4 +99,20 @@ public final synchronized void removeAnySubscription(ConnectableDevice device) {
     protected ResponseListener<R> getDefaultResponseListener() {
         return defaultResponseListener;
     }
+
+    /**
+     * A convenience method that calls device.hasCapability, but logs a message if the result is false.
+     *
+     * @param device the webos tv
+     * @param logger logger of the actual implementation to ensure the user can relate the channel
+     * @param capability the capability to check
+     *
+     */
+    protected boolean hasCapability(ConnectableDevice device, Logger logger, String capability) {
```

Comment:
Why do you pass a logger here, imho you rather use the logger from this class.

Reply:
Added this.getClass to log statement